docs/source/markdown/podman-system-df.1.md

```diff
@@ -14,6 +14,19 @@ Show podman disk usage
 
 Pretty-print images using a Go template or JSON. This flag is not allowed in combination with **--verbose**
 
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder**           | **Description**                     |
+| ------------------------- | ----------------------------------- |
+| .Active                   | Indicates whether volume is in use  |
+| .RawReclaimable           | Raw reclaimable size of each Type   |
+| .RawSize                  | Raw size of each type               |
+| .Reclaimable              | Human reclaimable size of each Type |
+| .Size                     | Human size of each type             |
+| .Total                    | Total items for each type           |
+| .Type                     | Type of data                        |
```

Comment:
"Human size" reads weirdly (to me). I like the parenthesized description you use elsewhere, would it make sense to write this table in that form?
```
.RawReclaimable | Reclaimable size (bytes)
.Reclaimable | Reclaimable size (human-readable)
...
```